storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -1031,6 +1033,13 @@ struct my_option xb_client_options[] = {
      (uchar *)&opt_lock_ddl, (uchar *)&opt_lock_ddl, 0, GET_BOOL, NO_ARG, 1, 0,
      0, 0, 0, 0},
 
+    {"orphan-check", OPT_ORPHAN_CHECK,
+     "During backup phase PXB only copies InnoDB tables who enteries matches "
+     "INFORMATION_SCHEMA.INNODB_TABLESPACES. use this option to disable this "
+     "validation and copy all ibd present in database ",
+     (uchar *)&opt_orphan_check, (uchar *)&opt_orphan_check, 0, GET_BOOL,
+     NO_ARG, 1, 0, 0, 0, 0, 0},
```

Comment:
bug fix doesn't warrant an option. I don't believe we should add this option.
I see that you are doing this check with --lock-ddl=on. So it is safe.

Reply:
removed option.